[2025-10-20T23:36:40.158+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T23:36:40.184+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_and_predict_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:40.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_and_predict_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T23:36:40.192+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T23:36:40.208+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_and_predict_task> on 2025-10-19 00:00:00+00:00
[2025-10-20T23:36:40.219+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=245) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T23:36:40.221+0000] {standard_task_runner.py:63} INFO - Started process 258 to run task
[2025-10-20T23:36:40.220+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_TaskFlow_FileIO', 'load_and_predict_task', 'scheduled__2025-10-19T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmp3losdxbc']
[2025-10-20T23:36:40.223+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask load_and_predict_task
[2025-10-20T23:36:40.301+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_TaskFlow_FileIO.load_and_predict_task scheduled__2025-10-19T00:00:00+00:00 [running]> on host 0074cd054b76
[2025-10-20T23:36:40.421+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_TaskFlow_FileIO' AIRFLOW_CTX_TASK_ID='load_and_predict_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-19T00:00:00+00:00'
[2025-10-20T23:36:40.425+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T23:36:40.427+0000] {airflow.py:95} INFO - Loading model /opt/***/dags/model/model.sav and predicting
[2025-10-20T23:36:40.441+0000] {logging_mixin.py:188} INFO - Optimal no. of clusters (elbow): 8
[2025-10-20T23:36:40.505+0000] {airflow.py:97} INFO - Prediction result: 5
[2025-10-20T23:36:40.514+0000] {python.py:237} INFO - Done. Returned value was: 5
[2025-10-20T23:36:40.520+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T23:36:40.620+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_TaskFlow_FileIO, task_id=load_and_predict_task, run_id=scheduled__2025-10-19T00:00:00+00:00, execution_date=20251019T000000, start_date=20251020T233640, end_date=20251020T233640
[2025-10-20T23:36:40.724+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T23:36:40.741+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-20T23:36:40.749+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
